environment.yml

```diff
@@ -1,9 +1,6 @@
 name: ivadomed_env
 channels:
   - conda-forge
-  - pytorch
 dependencies:
   - python>=3.7, < 3.9
   - pip
-  - pip:
-      - --requirement requirements.txt
```

Comment:
This PR is supposed to be about docs, but this changes the actual code's behaviour subtly.

Do we even want this? It's probably better to use the pytorch conda channel if we have it available isn't it? Or, at worst, it's irrelevant, because `pip --extra-index-url ...` will override/ignore conda's repos?